### PR TITLE
Quarter five pipeline - (Mag 16 -20)

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This covers only Magazine 16 - 20. Quarter five will cover upto Drama so2e03 so this needs to be updated with new drama seasons once the pipeline endpoint doc is updated.

### CODE REVIEW

File: code_schemes/s01mag16.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (lukechurch) [inline code_schemes/s01mag16.json]: Please can we change to Q5 everywhere we're using 'quarter'? Dates are generally in Q1, Q2 etc.

File: configurations/pipeline_config.json [hunk 2]
```diff
@@ -33,7 +33,13 @@
     "mag_s01e12_activation",
```
Comment (lukechurch) [inline configurations/pipeline_config.json]: Consider renaming to full_pipeline_config.json

File: configurations/quarter_five_pipeline_config.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (lukechurch) [inline configurations/quarter_five_pipeline_config.json]: q5_pipeline_config.json?

**Comment by IsaackMwenda** [comment]
@lukechurch PTAL

**Pull request merged**